tests/functional/test_home.py

```diff
@@ -0,0 +1,69 @@
+"""
+Test `pip home` command (pip.commands.home).
+"""
+
+from _pytest.monkeypatch import monkeypatch
+
+from mock import Mock, patch
+from pip.basecommand import ERROR, SUCCESS
+from pip.commands.home import HomeCommand
+
+
+mpatch = monkeypatch()
```

Comment:
You shouldn't need to import and instsantiate this manually, just add a `monkeypatch` argument to test functiont that need it.

Doing it this way means that stuff isn't rolled back correctly.